doc/topics/the-event-loop-timers-and-nexttick

```diff
@@ -0,0 +1,174 @@
+# Overview of the Event Loop, Timers, and `process.nextTick()`
+
+The Following diagram shows a simplified overview of the event loop's order of operations.
+
+       ┌───────────────────────┐
+    ┌─>│        timers         │
+    │  └──────────┬────────────┘
+    │  ┌──────────┴────────────┐
+    │  │   pending callbacks   │
+    │  └──────────┬────────────┘      ┌───────────────┐
+    │  ┌──────────┴────────────┐      │   incoming:   │
+    │  │         poll          │<─────│  connections, │
+    │  └──────────┬────────────┘      │   data, etc.  │
+    │  ┌──────────┴────────────┐      └───────────────┘
+    └──│     setImmediate      │
+       └───────────────────────┘
+
+note: each box will be referred to as a "phase" of the event loop.
+
+*There is a slight discrepancy between the Windows and the Unix/Linux
+implementation, but that's not important for this demonstration.  The most
+important parts are here.  There are actually seven or eight steps, but the
+ones we care about--ones that Node actually uses are these four.*
```

Comment:
libuv's event loop does have more steps than what is provided here. For example, `uv_handle_t`'s are not cleaned up until the (near) final phase of the event loop `uv__run_closing_handles()`. node has no hooks into this phase, so I don't deem it note worthy.

Or that if the event loop is run with `UV_RUN_ONCE` (which node does) then there's an additional step of updating and running timers. This is also not notable because the first step of the event loop after looping around is to also process timers.